app/models/work.rb

```diff
@@ -355,8 +355,24 @@ def uploads
     pre_curation_uploads_fast
   end
 
+  def pre_curation_s3_files
```

Comment:
@james it looks like you are trying to utilize local storage for development.  I'm not sure this will still work.  It would be better just to get the AWS keys from the staging server.